plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+        sample: "ansible-test-global-cluster"
+    global_cluster_resource_id:
+        description:
+        - The Amazon Web Services Region-unique, immutable identifier for the global database cluster.
+        type: str
+        sample: cluster-xxx
+    global_cluster_arn:
+        description:
+        - The Amazon Resource Name (ARN) for the global database cluster.
+        type: str
+        sample: "arn:aws:rds::xxx:global-cluster:ansible-test-global-cluster"
+    status:
+        description: The status of the DB cluster.
+        type: str
+        sample: available
+    engine:
+        description: The database engine of the DB cluster.
+        type: str
+        sample: aurora-postgresql
+    engine_version:
+        description: The database engine version.
+        type: str
+        sample: 14.8
+    storage_encrypted:
+        description: Whether the DB cluster is storage encrypted.
+        type: bool
+        sample: false
+    deletion_protection:
+        description:
+        - Indicates if the DB cluster has deletion protection enabled.
+          The database can't be deleted when deletion protection is enabled.
+        type: bool
+        sample: false
+    gloabl_cluster_members:
+        description:
+        - The list of primary and secondary clusters within the global database
+          cluster.
+        type: list
+        elements: dict
+        contains:
+            db_cluster_arn:
+                description: The Amazon Resource Name (ARN) for each Aurora DB cluster in the global cluster.
+                type: str
+                sample: arn:aws:rds:us-east-1:123456789012:cluster:ansible-test-primary
+            readers:
+                description: The Amazon Resource Name (ARN) for each read-only secondary cluster associated with the global cluster.
+                type: list
+                elements: str
+                sample: arn:aws:rds:us-east-2:123456789012:cluster:ansible-test-secondary
+            is_write:
+                description:
+                - Indicates whether the Aurora DB cluster is the primary cluster for the global cluster with which it is associated.
+                type: bool
+                sample: false
+            global_write_forwarding_status:
+                description: The status of write forwarding for a secondary cluster in the global cluster.
+                type: str
+                sample: disabled
+    failover_state:
+        description:
+        - A data object containing all properties for the current state of an in-process or
+          pending switchover or failover process for this global cluster (Aurora global database).
+        - This object is empty unless the SwitchoverGlobalCluster or FailoverGlobalCluster operation was called on this global cluster.
+        type: dict
+        contains:
+            status:
+                description:
+                - The current status of the global cluster. Possible values are as follows
```

Comment:
If this a sentence fragment or need to removed/completed?
```suggestion
                - The current status of the global cluster. 
```